.eslintrc

```diff
@@ -0,0 +1,113 @@
+{
+  "env": {
+    "browser": true,
+    "node": true
+  },
+
+  "rules": {
+    // Maybe enable?
```

Comment:
Some of these were suggested over in https://github.com/mochajs/mocha/issues/1359, some are more controversial